### ISSUE TITLE
Chosen Positions is not updated when editing a voter if it is empty.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#4]
```
What steps will reproduce the problem?
1. Edit a voter.
2. Move all data from Chosen Positions to Possible Positions
3. Press submit.

What is the expected output? What do you see instead?
Chosen Positions should now be empty but it still holds the previous data.


Please use labels and text to provide additional information.


```

Original issue reported on code.google.com by `waldemar...@gmail.com` on 11 Feb 2008 at 3:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `waldemar...@gmail.com` on 11 Feb 2008 at 3:37
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `toni...@gmail.com` on 16 Feb 2008 at 12:06
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2df9a52592.
```

Original comment by `waldemar...@gmail.com` on 4 Sep 2010 at 1:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `aicapa...@gmail.com` on 7 Feb 2011 at 9:19
- Changed state: **Verified**
